### PR TITLE
Restructure readme

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -17,39 +17,39 @@ future users of the library.
...
+- Maintained minor versions: 2 most recent
+
+Maintained Major Branches
+-------------------------
+
+.. csv-table::
+   :header: "Major", "Initial release date", "Supported until"
+   :widths: 5, 15, 12
+
+   `v5.x.x`_, `v5.2.0`_: 2017-03-31, 2017-05-01
```
Comment (ben-palmer-sociomantic) [inline README.rst]: I wonder about the supported until date here. I would keep it as TBA until we have a `v6.x.x` released.

Comment (ben-palmer-sociomantic) [reply README.rst]: or `TBA - 6 months after next major release`

Comment (ben-palmer-sociomantic) [reply README.rst]: Oh and the initial release date should be in 2016 :)

File: README.rst [hunk 2]
```diff
@@ -17,39 +17,39 @@ future users of the library.
 
 For more details on new features please refer to the
 `release notes <https://github.com/sociomantic-tsunami/libdrizzle-redux/releases>`_
+Support guarantees
```
Comment (ben-palmer-sociomantic) [inline README.rst]: Tiny thing but `guarantees` needs a capital G.

**Review by ben-palmer-sociomantic — COMMENTED** [review]
I had a couple of comments but generally looks ok.

File: README.rst [hunk 3]
```diff
@@ -65,9 +66,8 @@ Alternatively you can build and customize::
     make
     make install
 
-For more information about compiling libdrizzle redux please
-refer to ``compiling.rst`` in the ``./docs`` folder or build the documentation,
-cf. below.
+For more information about compiling **libdrizzle-redux** please
+refer to `compiling.rst`_ in the or build the documentation, cf. below.
```
Comment (ben-palmer-sociomantic) [inline README.rst]: Has something been removed between `in the` and `or build`?

**Review by ben-palmer-sociomantic — COMMENTED** [review]
Sorry I missed this on my last review.

File: README.rst (hunk 1 above)
Comment (andreas-bok-sociomantic) [reply README.rst]: > TBA - 6 months after next major release

Will change to this.

**Comment by andreas-bok-sociomantic** [comment]
Updated based on comments from ben

**Comment by ben-palmer-sociomantic** [comment]
LGTM

**Pull request merged**